tests/integration_tests/test_experiment.py

```diff
@@ -386,8 +390,8 @@ def test_experiment_image_dataset(
     }
 
     # create temporary name for train and test data sets
-    train_csv_filename = 'train_' + uuid.uuid4().hex[:10].upper() + '.csv'
-    test_csv_filename = 'test_' + uuid.uuid4().hex[:10].upper() + '.csv'
+    train_csv_filename = tmpdir+'train_' + uuid.uuid4().hex[:10].upper() + '.csv'
+    test_csv_filename = tmpdir+'test_' + uuid.uuid4().hex[:10].upper() + '.csv'
```

Comment:
In theory `tmpdir + 'test_' + ....' may or may not work depending if `tmpdir` has a trailing slash.  A better approach is to use the `os.path.join()` function to create the fully qualified path name.  This makes it platform independent.